src/utils/metrics.ts

```diff
@@ -0,0 +1,27 @@
+import * as mixpanelConfig from 'mixpanel-browser';
+import { sdk } from '../index';
+
+const mixpanel = mixpanelConfig.init(
+  'fb25742efb56d116b736515a0ad5f6ef',
```

Comment:
Can we extract this into a variable called `MIXPANELTOKEN` or something? Just feels a bit cleaner.